src/jit/alloc.cpp

```diff
@@ -1,411 +1,389 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
-/*****************************************************************************/
-
 
 #include "jitpch.h"
-#ifdef _MSC_VER
+
+#if defined(_MSC_VER)
 #pragma hdrstop
-#endif
-/*****************************************************************************/
+#endif // defined(_MSC_VER)
 
-/*****************************************************************************/
-void                allocatorCodeSizeBeg(){}
-/*****************************************************************************/
-#ifdef  DEBUG
-/*****************************************************************************/
+size_t ArenaAllocator::s_defaultPageSize = 0;
+ArenaAllocator* ArenaAllocator::s_pooledAllocator;
+ArenaAllocator::MarkDescriptor ArenaAllocator::s_pooledAllocatorMark;
+LONG ArenaAllocator::s_isPooledAllocatorInUse = 0;
 
-void    __cdecl     debugStop(const char *why, ...)
+bool ArenaAllocator::useHostAllocator()
```

Comment:
Since you're in here redoing everything, please add the function header comments per coding conventions.